packages/react/slider/src/Slider.tsx

```diff
@@ -117,7 +117,7 @@ const Slider = React.forwardRef((props, forwardedRef) => {
       const nextValues = getNextSortedValues(prevValues, nextValue, atIndex);
       if (hasMinStepsBetweenValues(nextValues, minStepsBetweenThumbs * step)) {
         valueIndexToChangeRef.current = nextValues.indexOf(nextValue);
-        return nextValues[atIndex] !== prevValues[atIndex] ? nextValues : prevValues;
+        return String(nextValues) === String(prevValues) ? prevValues : nextValues;
```

Comment:
I understand the change, but I can't wrap my head around why the previous way would create the keyboard bug yet this wouldn't. 🤔

Reply:
For anyone else following along, without this change we ended up with something like this:

```
[15,16,20] = atIndex = 0, valueIndexToChange = 0
[16,16,20] = atIndex = 0, valueIndexToChange = 0
[16,17,20] = atIndex = 0, valueIndexToChange = 1
```

That last array would then check `nextValues[0] !== prevValues[0]` (`16 !== 16`) which would be equal and return the `prevValues`.